docs/book/content/contributing/contributor_guide.md

```diff
@@ -125,11 +125,9 @@ situations, in which case other contributors are here to help.
    this, run the following command from the command line from inside
    the OG-USA/ogusa directory:
     ```
-     OG-USA/ogusa$ py.test
+     OG-USA/ogusa$  pytest -m "not needs_puf"
     ```
-   If the tests do not pass, try to fix the issue by using the
-   information provided by the error message. If this isn't possible
-   or doesn't work, we are here to help.
+   Note that running this full suite of tests may take close to 24 hours (depending on your hardware).  To run the small set of tests that run on each pull request (and take about 40 minutes), use  `pytest -m "not needs_puf and not local"`.  If the tests do not pass, try to fix the issue by using the information provided by the error message. If this isn't possible or doesn't work, we are here to help.
```

Comment:
@jdebacker, In the last line, I would change it to say "...or does not work, the core maintainers are available to help via GitHub issue."

Reply:
@jdebacker . An updated line was put here, but there is still a typo. See comment below. Marking this box resolved.